src/Microsoft.SqlTools.CoreServices/Microsoft.SqlTools.CoreServices.csproj

```diff
@@ -1,5 +1,4 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="../../Common.props" />

```

Comment:
Why is nothing importing common.props now? Don't we need settings from this?

Reply:
msbuild 15+ automatically imports Directory.Build.props and Directory.Build.targets files it finds in the current directory or a parent directory.